src/app/package-json.js

```diff
@@ -107,7 +108,10 @@ const packageJSON = (current, context) => {
       ]),
 
       'build:js:browserify': scriptify([
-        'browserify . -s %s -o dist/%s.js'
+        'browserify .',
+        '-t babelify',
+        '-s ' + context.functionName,
```

Comment:
it seems I had a previous version of the generator installed that was setting standalone to something like this: 
browserify -s src/stub.js